### ISSUE TITLE
Can this be used in a production app?

### ISSUE THREAD

**Issue by thedp** [icanzilb/MTLog#11]
A very useful lib. One question, will this pass App Store's review?

Thanks.

**Comment by icanzilb** [comment]
@thedp yes you can use it in production - MTLog is automatically removed in release builds.
